echopype/echodata/echodata.py

```diff
@@ -433,7 +440,7 @@ def mapping_get_multiple(mapping, keys, default=None):
                     mapping_get_multiple(
                         extra_platform_data,
                         ["water_level", "WATER_LEVEL"],
-                        np.ones(num_obs),
+                        default=np.ones(num_obs),
```

Comment:
@emiliom : Why is this `np.ones` instead of `np.zeros` or  use `np.nan` like in other variables?  It seems bizarre to default water level to be 1 m?

Reply:
I don't know. I didn't pay attention to that at all. The change I added in my PR was only stylistic -- in all the similar calls to `mapping_get_multiple`, the `default` parameter is used explicitly rather than implicitly (positionally).

Reply:
Ah I see, I somehow read this as adding a line, but you added the `default`.
My question should be for @imranmaj  :)

Reply:
I remember talking about this in a meeting, but I don't remember the exact reasoning; I made a PR specifically to make this change https://github.com/ngkavin/echopype/pull/3/files#diff-d35f9ea311a9dd316616bdd6291eb68fdb343808ce6ad80dc6b17402ec56db9aR563

Reply:
It was np.nan originally

Reply:
I wonder whether it was because in later operations, say for shipboard deployment, the true depth would be `water_level+range`, and if we have `np.nan` the outcomes would be NaN. I think though it should be `np.zeros` instead of `np.ones`